zerver/webhooks/trello/view/card_actions.py

```diff
@@ -114,8 +114,12 @@ def get_subject(payload):
 def get_body(payload, action_type):
     # type: (Mapping[str, Any], Text) -> Text
     message_body = ACTIONS_TO_FILL_BODY_MAPPER[action_type](payload, action_type)
+    template = TRELLO_MESSAGE_TEMPLATE
+    # if its a quote block, we don't need a period at the end
+    if message_body.endswith('~~~\n'):
+        template = TRELLO_MESSAGE_TEMPLATE.rstrip('.')
```

Comment:
Should we just remove the period from `TRELLO_MESSAGE_TEMPLATE`?  It seems brittle to strip off the period dynamically.  We can then explicitly append the period when we need it.  I'm not sure we ever actually need the period.

Reply:
Yeah, that's probably a good idea.  Or maybe just get rid of these templates altogether; they don't have a lot in them.

```
TRELLO_SUBJECT_TEMPLATE = u'{board_name}.'                                                                          
TRELLO_MESSAGE_TEMPLATE = u'{full_name} {rest}.'                                                                    
```

and aren't used in many places:

```
$ git grep TRELLO_MESSA
zerver/webhooks/trello/view/board_actions.py:from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
zerver/webhooks/trello/view/board_actions.py:    return TRELLO_MESSAGE_TEMPLATE.format(full_name=creator, rest=message_body)
zerver/webhooks/trello/view/card_actions.py:from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
zerver/webhooks/trello/view/card_actions.py:    return TRELLO_MESSAGE_TEMPLATE.format(full_name=creator, rest=message_body)
zerver/webhooks/trello/view/templates.py:TRELLO_MESSAGE_TEMPLATE = u'{full_name} {rest}.'
```